README.md

```diff
@@ -98,6 +98,19 @@ The pipe's capacity for stdout/stderr can be modified by specifying the `agent.c
 to the guest kernel command line. For example, `agent.container_pipe_size=2097152` will set the stdout and stderr
 pipes to 2097152 bytes.
 
+## Uevent Netlink Socket Receive Buffer Size
+
+When hotplugging a huge size memory into the Kata VM, the kernel in the VM will produce a lot of memory object add
+uevents and send all these uevents to kata-agent by netlink socket. However, default netlink socket recv buffer
```

Comment:
- Can you specify either "Kata agent" or `kata-agent` (here and below).
- I think the spell checker will complain about "recv" so you need to either specify "receive" or put the word in backticks: `recv`.